### ISSUE TITLE
Midi and Mus playback stopped with 3.1.1.0

### ISSUE THREAD

**Issue by EffinghamHuffnagel** [sirjuddington/SLADE#538]
**SLADE Version**: (3.1.1.0)
**Operating System**: (Windows 10 Pro, 64-bit)

**Issue Details**:
Midi and Mus playback worked fine in 3.1.0.4 (although you had to hit the play button twice before Midi would play. Never understood that.)

As of 3.1.1.0, hitting the play button does nothing for Midi and Mus entries. Ogg and Mp3 entries still play correctly.

I've just been doing the update and manually replacing files (Just updated to 3.1.1.2). Does a full install do something differently? I thought sure that somebody else would have mentioned it before now.

**Comment by Monsterovich** [comment]
Timidity or fluidsynth. What do you use?

**Comment by EffinghamHuffnagel** [comment]
Crap. I completely forgot I actually posted this until I downloaded the new Slade version. Very sorry.

Under Slade Preferences-Audio, nothing is checked or filled in, so it must be using fluidsynth by default.

If it helps at all, in my ZDoom sound options (ver 4/8/16) I have to use "Sound System". Neither fluidsynth nor timidity will play any music there either.

**Comment by sirjuddington** [comment]
Closing - looks like a system issue

Feel free to reopen the issue if needed

**Comment by EffinghamHuffnagel** [comment]
Crap. Sorry. I forgot this was still open. I meant to ask you to close it a long time ago. I finally did a full install of a newer version and looked at the install log for errors. That's when I figured out that older versions of Slade came with its own soundfont. I didn't realize I needed to install my own with newer versions. Found one I liked and everything works. Sorry again.


Thanks.


Lars Timpa

>
